doc(chart): update README.md of chart (#8398)
longhorn/longhorn: chart/README.md

| ingress.host | string | `"sslip.io"` | Hostname of the Layer 7 load balancer. |
| ingress.ingressClassName | string | `nil` | IngressClass resource that contains ingress configuration, including the name of the Ingress controller. ingressClassName can replace the kubernetes.io/ingress.class annotation used in earlier Kubernetes releases. |
| ingress.path | string | `"/"` | Default ingress path. You can access the Longhorn UI by following the full ingress path {{host}}+{{path}}. |

> | ingress.pathType | string | `"ImplementationSpecific"` | Ingress path type. Defaults to "ImplementationSpecific" for backward-compatibility reasons. |

Reviewer: ```suggestion
| ingress.pathType | string | `"ImplementationSpecific"` | Ingress path type. To maintain backward compatibility, the default value is "ImplementationSpecific". |
```

Author: Modified here and also updated chart/values.yaml. Thanks.
